src/core/mac/data_poll_sender.hpp

```diff
@@ -172,7 +172,7 @@ class DataPollSender : public InstanceLocator
      * pending frame.
      *
      */
-    void CheckFramePending(Mac::RxFrame &aFrame);
```

Comment:
Please modify comments to reflect the new usage of this function if you want to use it this way.